addons/udes_security/models/res_users.py

```diff
@@ -99,3 +99,20 @@ def _check_is_admin(self, values, group):
                     )
                     % group.name
                 )
+
+    @api.model_cr_context
+    def _auth_timeout_enabled(self):
+        """ Pluggable method to check if session timeout is enabled """
+        auth_timeout_enabled_parameter = self.env["ir.config_parameter"].get_param("inactive_session_time_out_enabled")
+        auth_timeout_enabled = auth_timeout_enabled_parameter == "True"
+        return auth_timeout_enabled
```

Comment:
Could directly return `auth_timeout_enabled_parameter == "True"`

Reply:
Yeah, was thinking it might make it a bit clearer rather than returning the expression. But its a very simple expression so will amend.

Reply:
Updated.